src/app/reporting/SynchronizedReportSchedulerImpl.cpp

```diff
@@ -0,0 +1,249 @@
+/*
+ *
+ *    Copyright (c) 2023 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include <app/InteractionModelEngine.h>
+#include <app/reporting/SynchronizedReportSchedulerImpl.h>
+#include <lib/support/logging/CHIPLogging.h>
+
+namespace chip {
+namespace app {
+namespace reporting {
+
+using Seconds16       = System::Clock::Seconds16;
+using Milliseconds32  = System::Clock::Milliseconds32;
+using Timeout         = System::Clock::Timeout;
+using Timestamp       = System::Clock::Timestamp;
```

Comment:
could this be replaced with a `using namespace System::Clock;` to avoid having aliases for all of them?